core/src/main/kotlin/io/customer/sdk/core/di/DiGraph.kt

```diff
@@ -133,18 +133,4 @@ abstract class DiGraph {
         overrides.clear()
         singletons.clear()
     }
-
-    // TODO: Remove deprecated functions after all usages are removed.
-    @Deprecated("Use overrideDependency<Dependency> instead", ReplaceWith("overrideDependency<Dependency>(value)"))
-    fun <Dependency : Any> overrideDependency(dependency: Class<Dependency>, value: Dependency) {
```

Comment:
Removed these as they were deprecated and are no longer used by any DI graph in the SDK